### PR TITLE
Fixed error if e.data.type was undefined

### PR DESCRIPTION
**Pull request by JamieRez**
Line 8937

### CODE REVIEW

File: dist/msrcrypto.js [hunk 1]
```diff
@@ -8934,7 +8934,7 @@ var workerManager = (function () {
 
             // Send the results to the operation object and it will fire
             //   it's onCompleted event.
-            if (op && e.data.type !== "process") {
+            if (op && e.data && e.data.type !== "process") {
```
Comment (JamieRez) [inline dist/msrcrypto.js]: Here's what I changed.

Comment (JamieRez) [reply dist/msrcrypto.js]: Without it, my react-native app crashes if a user types in the wrong password.

**Comment by kevlened** [comment]
Thanks for the contribution! Please undo the formatting changes; it'd be easier to review.

**Comment by JamieRez** [comment]
Actually, it seems I'm still having problems with authenticating even after this change. I think this should be closed. Disregard Everything.

**Comment by kevlened** [comment]
Ha, 👌

**Pull request closed without merging**